scripts/map.js

```diff
@@ -407,82 +424,65 @@ function startLocationDotRotation() {
 	};
 
 	if (isIOS) {
+		// create function for prompting user
+		const promptUserForPermission = () => {
+			createCustomYesNoPrompt(
+				"Nos dispositivos Apple, é necessária permissão do utilizador para aceder à bússola do dispositivo.",
+				() => {
+					// User clicked OK
+					DeviceOrientationEvent.requestPermission()
+						.then(response => {
+							if (response === "granted") {
+								window.addEventListener(
+									"deviceorientation",
+									e => {
+										requestAnimationFrame(() => handler(e));
+									},
```

Comment:
Não sei se isto está bem formatado por isso formata depois com o prettier antes do merge sff
```suggestion
									e => requestAnimationFrame(() => handler(e)),
```